build/test-requirements.txt

```diff
@@ -2,10 +2,10 @@ flake8
 flatbuffers==1.12
 # For now, we pin the numpy version here
 numpy>=1.16
-mypy==0.790
+mypy==0.902
```

Comment:
Please do not mix different kinds of changes in the same PR. Bumping the mypy version and improving the docs are both fine things to do, but please do them in separate PRs that try to do one thing only.

Reply:
This still need to be addressed. I would just revert the mypy change from this PR and send it as a separate PR.

Please also squash your commits and fix the title and commit comments to reflect that this is a doc-only change.